src/lib/postMessageToDiscord.ts

```diff
@@ -0,0 +1,24 @@
+import SemanticReleaseError from "@semantic-release/error";
+import fetch from "node-fetch";
+
+export async function postMessageToDiscord(message, discordWebhook) {
+  try {
+    const response = await fetch(discordWebhook, {
+      method: "post",
+      headers: {
+        "Content-Type": "application/json; charset=utf-8",
+      },
+      body: JSON.stringify(message),
+    })
+    const { status } = response
+    if (!String(status).startsWith('20')) {
+      const responseContent = await response.json()
+      throw new SemanticReleaseError(
```

Comment:
Why do you throw here ? you have a 200 status code and a json message at this point.